docs/configuring.md

```diff
@@ -703,6 +703,39 @@ questions with default answers.
     copier -fd 'user_name=Manuel Calavera' copy template destination
     ```
 
+### `data_file`
+
+-   Format: `str`
+-   CLI flags: `--data-file`
+-   Default value: N/A
+
+As an alternative you can also pass the path to a YAML file that contains your data.
+Passing a data file is only supported through the CLI.
+
+This cannot be defined in `copier.yml`, where its equivalent would be just normal
+questions with default answers.
+
+!!! example
+
+    Example CLI usage with a YAML file containing data:
+
+    ```yaml
```

Comment:
How about adding the file name `input.yml` as the title, so there is a better connect with the CLI call below?

```suggestion
    ```yaml
```

Reply:
Addressed